blockchain/state_migration.go

```diff
@@ -84,9 +84,12 @@ func (bc *BlockChain) concurrentRead(db *statedb.Database, quitCh chan struct{},
 	}
 }
 
-func (bc *BlockChain) migrateState(rootHash common.Hash) error {
+func (bc *BlockChain) migrateState(rootHash common.Hash) (returnErr error) {
```

Comment:
The return of `migrateState` is not used in anywhere.

Reply:
Right. But, in the defer function, it can be used well.
Is it ok?